#include <iostream>

class ExceptionWhenEndOfCalculation : public std::exception {
};
